build/contracts/Proxied.json

```diff
@@ -0,0 +1,983 @@
+{
```

Comment:
Is this needed? It can be imported like this:
`import Proxied from '@gnosis.pm/util-contracts/build/contracts/Proxied.json'`